core/core-services-impl/src/main/java/com/sflpro/notifier/services/notification/impl/push/PushNotificationRecipientServiceImpl.java

```diff
@@ -72,6 +73,18 @@ private void assertSearchParameters(final PushNotificationRecipientSearchParamet
         Assert.notNull(searchParameters, "Search parameters should not be null");
     }
 
+    /*TODO chk with Mr. Smith*/
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)